frontend/assets/icons/viewer/zoom_in.svg.tsx

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
When zoom in (or out) is disabled, I still get the tooltip and the "+" (or "-") changes colour
![image](https://github.com/3drepo/3drepo.io/assets/43547703/2b8f9244-305f-4420-b5c3-5e12e4b78f6a)

Reply:
- I don't see why the tooltip shouldn't be there.  We already have that in other places. 
- But the centre should change colour yes